docs/dev/adding_instruments.rst

```diff
@@ -693,3 +693,29 @@ Another use case involves maintaining compatibility between instruments with com
 
 In the above example, :code:`MultimeterA` and :code:`MultimeterB` use a different command to read the voltage, but the rest of the behaviour is identical.
 :code:`MultimeterB` can be defined subclassing :code:`MultimeterA` and just implementing the difference.
+
+
+Naming derived instruments
+**************************
+
+When deriving an instrument from an existing implementation, you might want to give separate names to any derived instrument. Two possible schemes for this are the following: 
```

Comment:
```suggestion
When deriving an instrument from an existing implementation, the derived instrument typically has a different name (e.g. another model number).
In addition, a user might want to give instantiated instruments setup-specific names (e.g. `Helmholtz_coil_driver` instead of `GenericPSU123`), especially if there is more than one device of the same make in a setup.
To accomodate these needs, there are two similar ways: 
```